cairo/programs/os.cairo

```diff
@@ -87,6 +87,7 @@ func apply_transactions{
 
     // Validate nonce
     with_attr error_message("Invalid nonce") {
+        assert_le(tx.signer_nonce, 2 ** 64 - 2);
```

Comment:
Note this fixes is handled differently than in Kakarot: 
Instead of doing the fix in the interpreter this is done in the transaction validation to match EELS.
https://github.com/ethereum/execution-specs/blob/1adcc1bfe774798bcacc685aebc17bd9935078c3/src/ethereum/cancun/fork.py#L377